src/responder/common/cache_req/plugins/cache_req_user_by_id.c

```diff
@@ -31,7 +31,7 @@ cache_req_user_by_id_create_debug_name(TALLOC_CTX *mem_ctx,
                                        struct cache_req_data *data,
                                        struct sss_domain_info *domain)
 {
-    return talloc_asprintf(mem_ctx, "UID:%d@%s", data->id, domain->name);
+    return talloc_asprintf(mem_ctx, "UID:%"SPRIgid"@%s", data->id, domain->name);
```

Comment:
This one is for `UID`. So it should be `SPRIuid`

But typ of `data->id` is `uint32_t` and not `gid_t/uid_t/id_t`.
So please check man inttypes.h for right format string for `uint32_t`